x/supply/alias.go

```diff
@@ -0,0 +1,51 @@
+// nolint
+// autogenerated code using github.com/rigelrozanski/multitool
+// aliases generated for the following subdirectories:
+// ALIASGEN: github.com/cosmos/cosmos-sdk/x/supply/keeper
+// ALIASGEN: github.com/cosmos/cosmos-sdk/x/supply/types
+// ALIASGEN: github.com/cosmos/cosmos-sdk/x/supply/exported
+package supply
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/supply/exported"
+	"github.com/cosmos/cosmos-sdk/x/supply/keeper"
+	"github.com/cosmos/cosmos-sdk/x/supply/types"
+)
+
+const (
+	ModuleName   = types.ModuleName
+	StoreKey     = types.StoreKey
+	RouterKey    = types.RouterKey
+	QuerierRoute = types.QuerierRoute
+	Holder       = types.Holder
+	Minter       = types.Minter
+	Burner       = types.Burner
+)
+
+var (
+	// functions aliases
+	RegisterInvariants  = keeper.RegisterInvariants
+	AllInvariants       = keeper.AllInvariants
+	TotalSupply         = keeper.TotalSupply
+	NewKeeper           = keeper.NewKeeper
+	NewModuleAccount    = types.NewModuleAccount
+	RegisterCodec       = types.RegisterCodec
+	NewGenesisState     = types.NewGenesisState
+	DefaultGenesisState = types.DefaultGenesisState
+	NewSupply           = types.NewSupply
+	DefaultSupply       = types.DefaultSupply
+
+	// variable aliases
+	DefaultCodespace = keeper.DefaultCodespace
+	ModuleCdc        = types.ModuleCdc
+)
+
+type (
+	Keeper         = keeper.Keeper
+	ModuleAccount  = types.ModuleAccount
+	AccountKeeper  = types.AccountKeeper
+	BankKeeper     = types.BankKeeper
+	GenesisState   = types.GenesisState
+	Supply         = types.Supply
+	ModuleAccountI = exported.ModuleAccountI
```

Comment:
We shouldn't be aliasing `exported`. The motivation for exported is to import it directly.